build.gradle

```diff
@@ -147,3 +148,28 @@ jacocoTestReport {
         xml.required = true
     }
 }
+
+diffCoverageReport {
+    diffSource { // Only one source may be specified
+        // This requires files to have been committed
+        // https://github.com/form-com/diff-coverage-gradle/issues/73
+        git.compareWith 'refs/remotes/origin/main'
+
+        // To compare uncommitted files, uncomment the below:
+        // file = 'build/tmp/git.diff'
+        // And then run:
+        //   git diff refs/remotes/origin/main > build/tmp/git.diff
+        //   ./gradlew diffCoverage
+    }
+
+    // View report at build/reports/jacoco/diffCoverage/html/index.html
+    reports {
+        html = true
+    }
+
+    violationRules {
```

Comment:
How about?
```
failOnViolation = true

Reply:
Sure, which would be the default if I deleted that block.    If we do want to fail we should decide what the required threshold for failure is.

Given that this is just a tool for developers to locally check before submitting a PR, the actual result is less important.  I left it as-is because locally I've wrapped it in a script and the failure would require me to add an `|| true` to make sure my script finishes to report missing coverage lines.  (I may submit that script later!)